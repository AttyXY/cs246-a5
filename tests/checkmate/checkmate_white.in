setup
+ K a8
+ k h1
+ r b1
+ r b2
= black
done
game human human
move b1 a1
