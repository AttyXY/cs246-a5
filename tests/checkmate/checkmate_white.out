WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  K-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  --------
   abcdefgh

8  K-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  -------k
   abcdefgh

8  K-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  -r-----k
   abcdefgh

8  K-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  -r------
1  -r-----k
   abcdefgh

STARTING GAME!

8  K-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  -r------
1  -r-----k
   abcdefgh

8  K-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  -r------
1  r------k
   abcdefgh

White is in check.
Checkmate! Black wins!
Final Score:
White: 0
Black: 1
THANKS FOR PLAYING CHESS!
